bindings/python/htcondor2/_negotiator.py

```diff
@@ -1 +1,180 @@
+from typing import List
 
+from ._common_imports import (
+    classad,
+    Collector,
+    DaemonType,
+)
+
+from .htcondor2_impl import (
+    _negotiator_command,
+    _negotiator_command_return,
+    _negotiator_command_user,
+    _negotiator_command_user_return,
+    _negotiator_command_user_value,
+)
+
+
+class Negotiator():
+
+    def __init__(self, location : classad.ClassAd = None):
+        if location is None:
+            c = Collector()
+            location = c.locate(DaemonType.Negotiator)
+
+        if not isinstance(location, classad.ClassAd):
+            raise TypeError("location must be a ClassAd")
+
+        self._addr = location['MyAddress']
+        # We never actually use this for anything.
+        # self._version = location['CondorVersion']
+
+
+    def deleteUser(self, user : str) -> None:
+        if not isinstance(user, str):
+            raise TypeError("user must be a string")
+        if '@' not in user:
+            # This was HTCondorValueError in version 1.
+            raise ValueError("You must specify the submitter (user@uid.domain)")
```

Comment:
There has to be a better way than doing this in every method, and even better still than calling a function each time... 🤔 nothing comes to mind immediately but I might have some research to do...

Reply:
Please let me know if you find anything.  I don't often want to use macros, but this would be a good case.